content/tutorials/react-setup.md

```diff
@@ -96,7 +97,7 @@ async componentDidMount() {
     parityUri: "http://localhost:8545",
     secretStoreUri: "http://localhost:12001"
   })
-  console.log("Finished loading contracts!")
+  console.log("Finished loading contracts.")
```

Comment:
If you're going to change that here then you should change it in all the other places too.

Reply:
I actually did everywhere I think